src/com_tjnotifications/admin/views/notifications/tmpl/default.php

```diff
@@ -15,6 +15,7 @@
 use \Joomla\CMS\HTML\HTMLHelper;
 use \Joomla\CMS\Router\Route;
 use \Joomla\CMS\Uri\Uri;
+use Joomla\CMS\Layout\LayoutHelper;
```

Comment:
in all files that you change remove `\` at the start

Reply:
resolved